lib/ui/dropdown/dropdown.js

```diff
@@ -87,6 +88,12 @@
         }
       });
 
+      if(angular.isDefined($attrs.multiple) && $element.get(0).tagName.toLowerCase() !== 'select') {
```

Comment:
create a helper function in the controller that test for remote multiple scenarios